site/content/docs/5.2/components/offcanvas.md

```diff
@@ -27,7 +27,7 @@ Offcanvas is a sidebar component that can be toggled via JavaScript to appear fr
 
 Below is an offcanvas example that is shown by default (via `.show` on `.offcanvas`). Offcanvas includes support for a header with a close button and an optional body class for some initial `padding`. We suggest that you include offcanvas headers with dismiss actions whenever possible, or provide an explicit dismiss action.
 
-{{< example class="bd-example-offcanvas p-0 bg-light overflow-hidden" >}}
+{{< example class="bd-example-offcanvas p-0 bg-body-secondary overflow-hidden" >}}
```

Comment:
Might be conflicting with my previous comments, but here we replace `.bg-light` with `.bg-body-secondary`. So we change a little bit the behavior compared to what's in our `main` branch. It would be rather `.bg-body-tertiary` (that seems to work in dark mode as well). Is there a specific reason to use `.bg-body-secondary` here instead of `.bg-body-tertiary`?

Reply:
Nah, this is a good thing to call out. Changing.